### ISSUE TITLE
ASM violation is not logging the request_s value

### ISSUE THREAD

**Issue by danielcayer** [F5Networks/f5-appsvcs-extension#460]
### Environment
 * Application Services Version: 3.27.0
 * Telemetry Streaming Version: 1.19.0
 * BIG-IP Version: 15.1.0.2

### Summary
I followed these instructions (https://devcentral.f5.com/s/articles/Deploying-BIG-IP-Telemetry-Streaming-with-Azure-Sentinel-as-its-consumer) and was able to successfully configure my BIG-IP to log violations to Azure Sentinel. The logged data correctly included the ASM "request_s" field, which contains the http host value I expected.
The following day, however, when I resumed working on this configuration, I found that the request_s field value was now empty in the Azure Log. I tried redeploying the TS configuration and rebooting the BIG-IP, but the request_s field remains empty. And I observed the the following error when I try re-deploying the AS3 declaration (I observe no errors when I re-deploy the TS declaration):

            "message": "declaration failed",
            "response": "01070110:3: Node address '/Common/Shared/255.255.255.254' is referenced by a member of pool '/Common/Shared/telemetry'.",

### Steps To Reproduce
1. Submit the following declaration:
```json
{ 
    "class": "ADC", 
    "schemaVersion": "3.10.0", 
    "remark": "Example depicting creation of BIG-IP module log profiles", 
    "Common": { 
        "Shared": { 
            "class": "Application", 
            "template": "shared", 
            "telemetry_local_rule": { 
                "remark": "Only required when TS is a local listener", 
                "class": "iRule", 
                "iRule": "when CLIENT_ACCEPTED {\n node 127.0.0.1 6514\n}" 
           }, 
            "telemetry_local": { 
                "remark": "Only required when TS is a local listener", 
                "class": "Service_TCP", 
                "virtualAddresses": [ 
                    "255.255.255.254" 
               ], 
                "virtualPort": 6514, 
                "iRules": [ 
                    "telemetry_local_rule" 
               ] 
           }, 
            "telemetry": { 
                "class": "Pool", 
                "members": [ 
                   { 
                        "enable": true, 
                        "serverAddresses": [ 
                            "255.255.255.254" 
                       ], 
                        "servicePort": 6514 
                   } 
               ], 
                "monitors": [ 
                   { 
                        "bigip": "/Common/tcp" 
                   } 
               ] 
           }, 
            "telemetry_hsl": { 
                "class": "Log_Destination", 
                "type": "remote-high-speed-log", 
                "protocol": "tcp", 
                "pool": { 
                    "use": "telemetry" 
               } 
           }, 
            "telemetry_formatted": { 
                "class": "Log_Destination", 
                "type": "splunk", 
                "forwardTo": { 
                    "use": "telemetry_hsl" 
               } 
           }, 
            "telemetry_publisher": { 
                "class": "Log_Publisher", 
                "destinations": [ 
                   { 
                        "use": "telemetry_formatted" 
                   } 
               ] 
           }, 
            "telemetry_traffic_log_profile": { 
                "class": "Traffic_Log_Profile", 
                "requestSettings": { 
                    "requestEnabled": true, 
                    "requestProtocol": "mds-tcp", 
                    "requestPool": { 
                        "use": "telemetry" 
                   }, 
                    "requestTemplate": "event_source=\"request_logging\",hostname=\"$BIGIP_HOSTNAME\",client_ip=\"$CLIENT_IP\",server_ip=\"$SERVER_IP\",http_method=\"$HTTP_METHOD\",http_uri=\"$HTTP_URI\",virtual_name=\"$VIRTUAL_NAME\",event_timestamp=\"$DATE_HTTP\"" 
               } 
           }, 
            "telemetry_security_log_profile": { 
                "class": "Security_Log_Profile", 
                "application": { 
                    "localStorage": false, 
                    "remoteStorage": "splunk", 
                    "protocol": "tcp", 
                    "servers": [ 
                       { 
                            "address": "255.255.255.254", 
                            "port": "6514" 
                       } 
                   ], 
                    "storageFilter": { 
                        "requestType": "illegal-including-staged-signatures" 
                   } 
               }, 
                "network": { 
                    "publisher": { 
                        "use": "telemetry_publisher" 
                   }, 
                    "logRuleMatchAccepts": false, 
                    "logRuleMatchRejects": true, 
                    "logRuleMatchDrops": true, 
                    "logIpErrors": true, 
                    "logTcpErrors": true, 
                    "logTcpEvents": true 
               } 
           } 
       } 
   } 
} 
```

2. Observe the following error response:
```json
{
    "id": "d5655bb3-e4b1-4765-b980-76f835cab89b",
    "results": [
        {
            "code": 200,
            "message": "success",
            "lineCount": 24,
            "host": "localhost",
            "tenant": "Common",
            "runTime": 3784
        },
        {
            "code": 422,
            "message": "declaration failed",
            "response": "01070110:3: Node address '/Common/Shared/255.255.255.254' is referenced by a member of pool '/Common/Shared/telemetry'.",
            "host": "localhost",
            "tenant": "Common",
            "runTime": 5184
        }
    ],
    "declaration": {
        "class": "ADC",
        "schemaVersion": "3.10.0",
        "remark": "Example depicting creation of BIG-IP module log profiles",
        "id": "autogen_1e973539-15d2-4bc3-8d97-24c1657421d1",
        "updateMode": "selective",
        "controls": {
            "archiveTimestamp": "2021-05-14T15:18:18.396Z"
        }
    }
}
```

### Steps To Reproduce
1. Submit the following declaration:
```json
same as above
```

2. Observe the following error response:
```json
same as above
```

### Expected Behavior
I expect to see the request_s value in Azure Sentinel as follows (the "Host" value of 10.1.10.35 is the one I really need to see in my Azure Logs):

GET /vulnerabilities/sqli/?id=%25%27+or+1%3D%271&Submit=Submit HTTP/1.1\r\nHost: 10.1.10.35\r\nUser-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:53.0) Gecko/20100101 Firefox/53.0\r\nAccept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8\r\nAccept-Language: en-US,en;q=0.5\r\nAccept-Encoding: gzip, deflate\r\nReferer: http://10.1.10.35/vulnerabilities/sqli/\r\nCookie: JSESSIONID=p7p6dft8j9vvnct49c6rht6sp1; security=low; TS019d830f=014ad15ee76f22f40e09e6bdd1d732565cc6bb6baada2eae63f6b427bb6bd069a4bcf25ed51faf0300025bcfbcc7b5eaa9787d020c79ae170688c560a236ab008d040b769d9de8db1522771d9d25931d1ae3c30f90\r\nDNT: 1\r\nConnection: keep-alive\r\nUpgrade-Insecure-Requests: 1\r\n\r\n

### Actual Behavior
A clear and concise description of what actually happens.
Please include any applicable error output.

The events (ASM violations) are being logged, but the request_s field remains blank in Azure Log.
![Screen Shot 2021-05-17 at 11 34 19 AM](https://user-images.githubusercontent.com/28303815/118516042-f51c7580-b703-11eb-9c55-c4f30fbae0c7.png)

**Comment by dstokesf5** [comment]
It looks like I am no longer able to reproduce the issue with the `/Common/Shared/255.255.255.254` node in version 3.29.0 of AS3. This version should be available early next week.

**Comment by dstokesf5** [comment]
This issue has been resolved by the 3.29.0 release.
